src/components/AddedFeature.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 
 const AddedFeature = props => {
+  //console.log(props);
   return (
     <li>
       {/* Add an onClick to run a function to remove a feature */}
-      <button className="button">X</button>
+      <button onClick={props.removeFeature} className="button">X</button>
```

Comment:
The remove functionality isn't currently working.  Check the different between this and how you have the `buyItem` click handler set up in `src/components/AdditionalFeature.js` (line: 8)